### PR TITLE
Support Django 1.11 in setup.py requirements

### PR DESCRIPTION
**Pull request by thallada**
@cahrens pointed this out. This repo now supports Django 1.11, so `setup.py` should reflect that too.

@edx/educator-dahlia

### CODE REVIEW

**Comment by thallada** [comment]
Looks like the Django 1.11 test was actually installing Django 1.10 before this change, so there's some failures here due to Django 1.11 breaking changes. I'll try and fix them up, but it looks like we are actually only supporting 1.10 at the moment.

**Comment by thallada** [comment]
Okay, @edx/educator-dahlia I fixed the Django 1.11 build.

File: edx_proctoring/backends/tests/test_software_secure.py [hunk 1]
```diff
@@ -87,6 +87,7 @@ def mock_response_error(url, request):  # pylint: disable=unused-argument
         }
     }
 )
+@patch('django.core.urlresolvers.reverse', MagicMock)
```
Comment (efischer19) [inline edx_proctoring/backends/tests/test_software_secure.py]: nice

File: setup.py [hunk 2]
```diff
@@ -60,6 +63,6 @@ def get_version(*file_paths):
     ],
     include_package_data=True,
     install_requires=[
-        "Django>=1.8,<1.11"
+        "Django>=1.8,<1.12"
```
Comment (cahrens) [inline setup.py]: We've standardized on <2.0 (see conversation on my milestones PR).

**Pull request merged**